src/runtime/runtime-test.js

```diff
@@ -1145,6 +1145,20 @@ describes.realWin('ConfiguredRuntime', {}, (env) => {
     ).to.throw();
   });
 
+  it('should pass enableDefaultMeteringHandler into EntitlementsManager during constuction and default to false', () => {
+    runtime = new ConfiguredRuntime(win, config);
+
+    expect(runtime.entitlementsManager().enableDefaultMeteringHandler_).to.be
```

Comment:
Same comment here re checking private variables in the test.

Reply:
Done